controllers/apply/reaching-communities/processor.test.js

```diff
@@ -9,14 +9,20 @@ const jsdom = require('jsdom');
 const { JSDOM } = jsdom;
```

Comment:
These test updates are based on a warning by the `jest` upgrade. This test was doing set up in an idiomatic way which Jest now flags as a warning.